core/src/components/button/button.ionic.scss

```
@@ -246,26 +267,32 @@
   font-size: globals.$ion-font-size-500;
 }
 
-:host(.button-small) ::slotted(ion-icon[slot="start"]) {
+:host(.button-small) ::slotted(ion-icon[slot="start"]),
+::slotted(ion-spinner[slot="start"]) {
   @include globals.margin-horizontal(null, globals.$ion-space-200);
 }
 
-::slotted(ion-icon[slot="start"]) {
+::slotted(ion-icon[slot="start"]),
+::slotted(ion-spinner[slot="start"]) {
   @include globals.margin-horizontal(null, globals.$ion-space-250);
 }
 
-:host(.button-large) ::slotted(ion-icon[slot="start"]) {
+:host(.button-large) ::slotted(ion-icon[slot="start"]),
+::slotted(ion-spinner[slot="start"]) {
   @include globals.margin-horizontal(null, globals.$ion-space-300);
 }
 
-:host(.button-small) ::slotted(ion-icon[slot="end"]) {
+:host(.button-small) ::slotted(ion-icon[slot="end"]),
+::slotted(ion-spinner[slot="end"]) {
   @include globals.margin-horizontal(globals.$ion-space-200, null);
 }
 
-::slotted(ion-icon[slot="end"]) {
+::slotted(ion-icon[slot="end"]),
+::slotted(ion-spinner[slot="end"]) {
   @include globals.margin-horizontal(globals.$ion-space-250, null);
 }
 
-:host(.button-large) ::slotted(ion-icon[slot="end"]) {
+:host(.button-large) ::slotted(ion-icon[slot="end"]),
+::slotted(ion-spinner[slot="end"]) {
```

```suggestion
:host(.button-large) ::slotted(ion-spinner[slot="end"]) {
```

:host(.button-small)